docs/integrations/containers-orchestration/opentelemetry/activemq-opentelemetry.md

```diff
@@ -0,0 +1,304 @@
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/containers-orchestration/activemq.png')} alt="icon" width="75"/>
+
+The **ActiveMQ** app is a unified logs and metrics app that helps you monitor the availability, performance, health, and resource utilization of your ActiveMQ messaging clusters. Preconfigured dashboards provide insight into cluster status, nodes, producers, consumers, destinations, resource utilization, message rates, and error logs.
+
+
+We use the OpenTelemetry collector for ActiveMQ metrics and logs collection.
+
+The diagram below illustrates the components of the ActiveMQ collection for each ActiveMQ broker node. OpenTelemetry collector runs on the same host as ActiveMQ broker, and uses the [JMX Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/jmxreceiver) to obtain ActiveMQ metrics, and the [Sumo Logic OpenTelemetry Exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/sumologicexporter) to send the metrics to Sumo Logic. ActiveMQ logs are sent to Sumo Logic through a [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/ActiveMQ-OpenTelemetry/ActiveMQ-OTel-Collection-architecture.png' alt="ActiveMQ OTel Collection architecture" />
+
+This App has been tested with following ActiveMQ versions:
+  * 5.17.4
+
+## Log Types and Metrics
+
+The Sumo Logic App for ActiveMQ assumes:
+
+- ActiveMQ app supports the below log format in log4j2.properties file
+
+  * [Audit Logs](https://activemq.apache.org/audit-logging):  Every management action made through JMX or Web Console management interface is logged in audit log files.
+
+    `appender.auditlog.layout.pattern=%d | %-5p | %m | %t%n`
+
+  * ActiveMQ Logs:
+
+    `appender.logfile.layout.pattern=%d | %-5p | %m | %c | %t%n%throwable{full}`
+
+
+- For a list of metrics that are collected and used by the app, see [ActiveMQ Metrics](https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/jmx-metrics/docs/target-systems/activemq.md).
+
+
+## Fields Creation in Sumo Logic for ActiveMQ
+
+Following are the [Fields](/docs/manage/fields/) which will be created as part of ActiveMQ App install if not already present.
+
+* `messaging.cluster.name`. User configured. Enter a name to uniquely identify your ActiveMQ cluster. This cluster name will be shown in the Sumo Logic dashboards.
+* `messaging.node.name`. Has value of `host name`.
+* `messaging.system`. Has fixed value of `activemq`.
+* `sumo.datasource`. Has fixed value of `activemq`.
+
+If Process Metrics are enabled it will also create [fields for JMX metrics](/docs/integrations/app-development/opentelemetry/jmx-opentelemetry/#fields-creation-in-sumo-logic-for-jmx)
```

Comment:
```suggestion
If Process Metrics are enabled it will also create [fields for JMX metrics](/docs/integrations/app-development/opentelemetry/jmx-opentelemetry#fields-creation-in-sumo-logic-for-jmx)
```

Reply:
This link is broken because the JMX article has not yet been published (PR #2734).